src/test/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/PostgresDirectoryDAOSpec.scala

```diff
@@ -11,7 +11,7 @@ import org.broadinstitute.dsde.workbench.sam.db.TestDbReference
 import org.broadinstitute.dsde.workbench.sam.db.tables.TosTable
 import org.broadinstitute.dsde.workbench.sam.matchers.TimeMatchers
 import org.broadinstitute.dsde.workbench.sam.model._
-import org.broadinstitute.dsde.workbench.sam.model.api.SamUserAttributes
+import org.broadinstitute.dsde.workbench.sam.model.api.{SamUser, SamUserAttributes}
```

Comment:
There are a lot of changes in this file, but they should not be scary.

1. scalafmt and adding types to some variables
2. adding the `assume` statement to each individual test because the tests were erroring for me locally when I only had it at the top of the suite.